src/event.rs

```diff
@@ -557,6 +557,7 @@ where
 							status: PaymentStatus::Succeeded,
 							lsp_fee_limits: None,
 							bolt11_invoice: None,
+							last_update: 0,
```

Comment:
Why not set to the current time instead of 0? (in all places)

Reply:
My thought was to make it `default`-initializable. Then I realized it is never initialized with `default()` but left it as is anyways.

Reply:
but in my opinion it does not make sense. Why would the last update time be 0? what if the payment is not updated?

Reply:
Just to clarify, I feel that `last_update` should not even be writeable: it should somehow be read-only, and it should be set automatically when a record is inserted/updated in the payment store. To implement it more or less properly, we would have to either provide a constructor method for `PaymentDetails` and set `last_update` there, or split `PaymentDetails` into a request (further split into create/update) and a response (only the responses would contain `last_update`).

However, all the better solutions for this problem require more changes to the code and more divergence from the upstream LDK repo. Therefore I decided to keep changes to the absolute minimum, especially since our fork is unlikely to be used by anyone else — and we only use the generated bindings, not the LDK framework itself. So I chose to initialize `last_update` with 0 everywhere, and to overwrite it with the appropriate value inside `PaymentStore` :)

Reply:
Ah ok, that makes sense. Thanks!